src/app/dataset-view/dataset.component.ts

```diff
@@ -210,49 +225,50 @@ export class DatasetComponent
         });
     }
 
-    public onClickNode(idDataset: string): void {
+    public onClickNode(node: any): void {
+        this.selectedDatasetName = node.label;
         this.datasetViewType = DatasetViewTypeEnum.overview;
-        this.onSelectDataset(idDataset);
+        this.onSelectDataset(node?.data.id);
     }
 
     public getDatasetNavigation(): DatasetNavigationInterface {
         return {
             navigateToOverview: () => {
-                this.navigationService.navigateToDatasetView(
-                    AppValues.defaultUsername,
-                    this.getDatasetId(),
-                    ProjectLinks.urlDatasetViewOverviewType,
-                );
+                this.navigationService.navigateToDatasetView({
+                    accountName: AppValues.defaultUsername,
```

Comment:
In all navigation functions like this we should not be loosing `accountName` from the URL